packages/react-composites/src/composites/CallWithChatComposite/components/PreparedDialpad.styles.ts

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { IButtonStyles, IModalStyles, Theme } from '@fluentui/react';
+
+import { DialpadStyles } from '@internal/react-components';
+
+/**
+ * @private
+ */
+export const themedDialpadModelStyle = (theme: Theme): Partial<IModalStyles> => ({
+  main: {
+    borderRadius: theme.effects.roundedCorner6,
+    padding: '1rem'
+  }
+});
+
+/**
+ * @private
+ */
+export const themedDialpadStyle = (isMobile: boolean, theme: Theme): Partial<DialpadStyles> => ({
+  root: {
+    padding: 0,
+    marginLeft: 0,
+    marginRight: 0,
+    maxWidth: '100%'
+  },
+  textField: {
+    root: {
+      borderBottom: '1px solid lightgrey'
+    },
+    field: {
+      backgroundColor: 'white',
+      fontSize: theme.fonts.large.fontSize,
+      padding: '1.063rem 0.5rem',
+      textAlign: isMobile ? 'center' : 'left',
+      paddingTop: 0
+    }
+  },
+  primaryContent: {
+    color: theme.palette.themeDarkAlt
+  }
+});
+
+/**
+ * @private
+ */
+export const themedCallButtonStyle = (theme: Theme): Partial<IButtonStyles> => ({
+  root: {
+    fontWeight: 600,
```

Comment:
Why are we enforcing a static fontWeight here?
Can we use `theme.fonts.medium.fontWeight`?